src/server.c

```diff
@@ -5600,17 +5600,63 @@ void redisOutOfMemoryHandler(size_t allocation_size) {
         allocation_size);
 }
 
+/* Callback for sdstemplate on proc-title-template. See redis.conf for 
+ * supported variables.
+ */
+static sds redisProcTitleGetVariable(const sds varname, void *arg)
+{
+    if (!strcmp(varname, "title")) {
+        return sdsnew(arg);
+    } else if (!strcmp(varname, "listen-addr")) {
+        return sdscatprintf(sdsempty(), "%s:%u",
```

Comment:
I think we wanted to "bug fix" the default behavior so that when there's no port, it'll implicitly show the Unix socket. 

Note that this would be mentioned in the commit comment and release notess